.npmignore

```diff
@@ -2,6 +2,7 @@ docs/
 lib/
 scripts/
 spec/
+testapp/
 
```

Comment:
should also have `built/spec` if your typescript files are being transpiled to `built`